semopenalex-app/data/templates/Template%3Ahttps%3A%2F%2Fsemopenalex.org%2Fclass%2FWork.html

```diff
@@ -1,76 +1,357 @@
+      }
+    }" template='{{>tmp}}'>
+    <template id='tmp'>
+      {{#each bindings}}
+        <div class="row cards">
+          <div class="card citations-chart">
+            <div class="publication_year_citations">
+              <div class="publication_year">
+                <div class="publication_year__label">Published in</div>
+                <div class="publication_year__value">
+                  {{publicationYear.value}}
+                </div>
+              </div>
+              <div class="citations">
+                <div class="citations__label">Citations</div>
+                <div class="citations__value">{{citedByCount.value}}</div>                  
+              </div>
+            </div>
+            <semantic-chart type="bar" query="
+              SELECT ?year ?citations WHERE {
+                <{{page-resource}}> <https://semopenalex.org/property/countsByYear> ?count .
+                ?count <https://semopenalex.org/property/year> ?year .
+                ?count <https://semopenalex.org/property/citedByCount> ?citations .
+              } ORDER BY ASC(?year)
+            " sets='[{"dataSetName": "Citations", "category": "citations", "x": "year", "y": "citations"}]'
+              styles='[{
+                "provider": "highcharts",
+                "style": {
+                  "series": [
+                    {"type": "column", "showInLegend": false}
+                  ]
+                }
+              }]'
+              style-highcharts='{
+                "chart": {
+                  "margin": [20, 0, 20, 40]
+                },
+                "legend": "hide"
+              }'
+              dimensions='{"height": 180}'
+            >
+            </semantic-chart>
           </div>
-            
+          <div class="card">
+            <div>
+              <div class="card__title">PUBLISHED IN</div>
+              <div class="publication-info">
+                {{#if publisher}}
+                  <semantic-link iri="{{publisher.value}}">{{publisher.value}},</semantic-link>
+                {{/if}}
+                {{#if volume}}
+                  <div>Volume {{volume.value}},</div>
+                {{/if}}
+                {{#if issue}}
+                  <div>Issue {{issue.value}},</div>
+                {{/if}}
+                {{#if publicationDate}}
+                  <div>{{publicationDate.value}}</div>
+                {{else}}
+                  <div>{{publicationYear.value}}</div>
+                {{/if}}
+              </div>
+              {{#if doi}}
+                <semantic-link iri="{{doi}}">{{doi.value}}</semantic-link>
+              {{/if}}
+            </div>
+          </div>
+        </div>
+      {{/each}}
+    </template>        
+  </semantic-query>
+</template-fragment>
 
-        
-          
+<template-fragment id="Concepts">
+  <div class="row concepts">
+  <semantic-query query='
+    SELECT ?concept (COUNT (distinct ?publication) as ?count) {
```

Comment:
Please select only `?concept`. We do not need `(COUNT (distinct ?publication) as ?count)`